documentation/browser-preview.workspace.adoc

```diff
@@ -0,0 +1,94 @@
+
+image::cypher-reference.gif[]
+
+== Richer table visualization
+
+The table visualization now employs reactive result fetching by default, meaning we’ll automatically limit the number of rows returned by a query regardless of any `LIMIT` statements you apply. This is useful for new and experienced users alike, where an unintentional omission of a `LIMIT` statement could result in the results returned could previously overwhelm Browser. The table now also employs virtualized rendering meaning that even if you do return a larger number of rows, we only render those in view to keep the application more responsive.
+
+Functionally there are improvements too. A new render for paths, relationships and nodes allows you to more readily discern labels/types and key/value pairs. By default we provide a compact render, but optionally allow graph objects to be pretty-printed for situations where ease of reading is more important than compactness.
+
+Simple result filtering is now also available, allowing you to perform simple text matches on returned results. This can be useful when it's simpler and quicker than modifying a Cypher statement to contain a WHERE clause.
+
+image::table-visualization.gif[]
+
+ 
+== More scalable graph visualization
+
+The graph visualization has been completely rebuilt using a new engine while maintaining the familiar layout algorithm from the ouutgoing graph visualization. This makes the visualization much more responsive and allows for a far larger number of nodes and relationships to be visualized than was previously possible.
+
+One important functional change is the visualization now only displays items you explicitly return in Cypher statements - no longer automatically connecting returned nodes with relationships that exist between them. This can still be achieved on-demand via a context menu in the visualization to “show all relationships”
+
+image::scalable-graph-visualization.gif[]
+
+
+== New history and quick search
+
+The familiar and convenient up/down arrow history feature that remains, but has been supplemented with a dedicated sidebar to review and manage history, allowing history to be selectively deleted or exported.
+
+For convenience, the history available in the sidebar is also searchable via a fuzzy search from the main editor, allowing you to quickly find a query in your history. This can be invoked from the … more menu on the main editor or via Ctrl+R
+
+image::history-quick-search.gif[]
+
+
+== Improved saved Cypher experience
+
+The “Favorites” experience from Browser is now “Saved Cypher” and provides a refreshed way of creating and managing Cypher. Among other things, you can now more readily re-order items among nested folders and cypher saved from the editors recalls the last folders used for convenience. 
```

Comment:
```suggestion
The “Favorites” experience from Browser is now “Saved Cypher” and provides a refreshed way of creating and managing Cypher. 
Among other things, you can now more readily re-order items among nested folders and Cypher saved from the editor recalls the last folders used for convenience. 
```
Not sure exactly what the last sentence is supposed to say, you can re-order items among nested folders and from Cypher saved from the editor, I get that, but the last bit "recalls the last folders used for convenience", I don't know what that refers to.